index.html

```diff
@@ -23,13 +24,21 @@ <h3>What's for Dinner?</h3>
             <label for="entire-meal">Entire Meal</label></li>
           </ul>
         </form>
-        <button id="lets-cook-button"><h4>LET'S COOK!</h4></button>
+        <button id="lets-cook-button">LET'S COOK!</button>
       </article>
       <article id="lets-cook-box">
         <p class="cookpot-image"><img src="./assets/cookpot.svg" alt="cooking pot with lid tilted off"></p>
         <p class="you-should-make hidden">You should make:</p>
-        <p class="output-recipe hidden">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-        <button class="clear-button hidden">CLEAR</button>
```

Comment:
Bye, bye Lorem ipsum.  Now it displays all of the deserts you could ever want instead.